sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>This is the Microsoft Azure Application Configuration Service client library</Description>
     <AssemblyTitle>Microsoft Azure.Data.AppConfiguration client library</AssemblyTitle>
     <Version>1.0.0-preview.5</Version>
-    <PackageTags>Microsoft Azure Application Configuration</PackageTags>
+    <PackageTags>Microsoft Azure Application Configuration, windowsazureofficial, azureofficial</PackageTags>
```

Comment:
It would be trival to put this in a shared property in https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Directory.Build.Data.props#L109 and use that property her. That would be all I expect we would do in the eng sys.

Reply:
What would happen if there are duplicates? Will Nuget simply ignore them?

And also, I think we would need to normalize how we separate the tags.

Reply:
Looks like nuget doesn't do anything special it simply creates a tag based on some delimiter, including space it looks like from https://www.nuget.org/packages/Azure.Storage.Blobs/ where you can see "Microsoft Azure Storage Blobs, Microsoft, Azure, Blobs, Blob, Storage, StorageScalable, azureofficial, windowsazureofficial" turned into multiple Microsoft, Azure, and Blobt ags.

Reply:
We can trim duplicates in a custom target.

Reply:
Or just `Contains("")` in conditions.